### ISSUE TITLE
This plugin does not seem to use same webpack config as vue-cli-service

### ISSUE THREAD

**Issue by zachmullen** [vue-styleguidist/vue-cli-plugin-styleguidist#17]
Steps to reproduce:

1. In a project, run `vue add styleguidist`
2. Edit `AppButton.vue`, changing the `handleClick(e)` method to `async handleClick(e)`.
3. Run `yarn styleguide:build`

Error message:

```
Warning: Cannot parse src/components/AppButton.vue: ReferenceError: regeneratorRuntime is not defined

It usually means that vue-docgen-api does not understand your source code or when using third-party libraries, try to file an issue here:
https://github.com/vue-styleguidist/vue-docgen-api/issues
```

This is strange since `yarn build` and `yarn serve` work fine and do not complain about the use of `async`. Explicitly adding `@babel/plugin-transform-runtime` to the babel config changes the error message, but does not fix it:

```
Warning: Cannot parse src/components/AppButton.vue: TypeError: _regenerator.default.mark is not a function
```

**Comment by zachmullen** [comment]
FWIW, when running `yarn styleguide:build`, it is finding the webpack config from vue-cli-service:

```
Loading webpack config from:
/Users/zach/dev/girder_web_components/node_modules/@vue/cli-service/webpack.config.js
```

It just seems to be doing something different with the babel configuration somewhere under the covers.

**Comment by elevatebart** [comment]
@zachmullen 

Can you checkout version 0.2.0 ? I believe this fixes your problem ;)

**Comment by zachmullen** [comment]
@elevatebart that does indeed fix the issue, this can be closed.

I do get a ton of warnings during build, all similar to the following:

```
./src/Components/HelloWorld.vue (./node_modules/vue-styleguidist/loaders/vuedoc-loader.js!./src/Components/HelloWorld.vue)
There are multiple modules with names that only differ in casing.
This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
Use equal casing. Compare these module identifiers:
* /Users/zach/dev/styleguidist-test/node_modules/vue-styleguidist/loaders/vuedoc-loader.js!/Users/zach/dev/styleguidist-test/src/Components/HelloWorld.vue
    Used by 1 module(s), i. e.
    /Users/zach/dev/styleguidist-test/node_modules/vue-styleguidist/loaders/styleguide-loader.js!/Users/zach/dev/styleguidist-test/node_modules/vue-styleguidist/lib/index.js
* /Users/zach/dev/styleguidist-test/node_modules/vue-styleguidist/loaders/vuedoc-loader.js!/Users/zach/dev/styleguidist-test/src/components/HelloWorld.vue
    Used by 1 module(s), i. e.
    /Users/zach/dev/styleguidist-test/node_modules/vue-styleguidist/loaders/styleguide-loader.js!/Users/zach/dev/styleguidist-test/node_modules/vue-styleguidist/lib/index.js
 @ ./src/Components/HelloWorld.vue (./node_modules/vue-styleguidist/loaders/vuedoc-loader.js!./src/Components/HelloWorld.vue)
 @ ./node_modules/vue-styleguidist/loaders/styleguide-loader.js!./node_modules/vue-styleguidist/lib/index.js
 @ ./node_modules/vue-styleguidist/lib/index.js
 @ multi ./node_modules/vue-styleguidist/lib/index
```

That's with a completely fresh project that only has this plugin installed, on MacOS. Is that a known issue?

**Comment by elevatebart** [comment]
it is a known issue, I will publish a fix tonight if I get to. 
You can fix it yourself by un-commenting the `components` line in the config. 
The standard config of vsg is weird.

Sorry for the inconvenience.
Bart

**Comment by zachmullen** [comment]
Cool, thanks for the info, and the fix!
